src/Query/OrderBy.php

```diff
@@ -15,6 +15,12 @@ class OrderBy
      */
     public function __construct(string $sort, string $direction)
     {
+        $direction = strtolower($direction);
+
+        if (! in_array($direction, ['asc', 'desc'], true)) {
+            throw new InvalidArgumentException('Order direction must be "asc" or "desc".');
```

Comment:
This needs to be imported. Currently throws a `Class "Statamic\Query\InvalidArgumentException" not found` error.